### PR TITLE
feat: analytics on dashboard

### PR DESCRIPTION
**Pull request by drew-harris**

### CODE REVIEW

**Comment by densumesh** [comment]
image

**Review by densumesh — CHANGES_REQUESTED** [review]
need to add back button

File: frontends/dashboard/src/analytics/components/charts/NoResultQueries.tsx [hunk 1]
```diff
@@ -73,18 +74,15 @@ export const NoResultQueries = (props: NoResultQueriesProps) => {
   console.log(notResultQuery.data);
```
Comment (SaraVieira) [inline frontends/dashboard/src/analytics/components/charts/NoResultQueries.tsx:74]: console log left behind

**Comment by SaraVieira** [comment]
Besides the back button mentioned the only think I have to add is that the button to open the analytics is still in the menu

Other than that this looks great!

**Review by densumesh — APPROVED** [review]

**Pull request merged**